cli/fossilize_replay.cpp

```diff
@@ -2994,6 +3027,16 @@ int main(int argc, char *argv[])
 	});
 	cbs.add("--num-threads", [&](CLIParser &parser) { replayer_opts.num_threads = parser.next_uint(); });
 	cbs.add("--loop", [&](CLIParser &parser) { replayer_opts.loop_count = parser.next_uint(); });
+	cbs.add("--pipeline-hash", [&](CLIParser &parser) {
+		const char *hash_str = parser.next_string();
+		char *end;
+		replayer_opts.pipeline_hash = strtoll(hash_str, &end, 16);
```

Comment:
Actually, shouldn't this should be strtoull?

Reply:
Yes, it should.  Good catch!  Fixed in the latest version.